Examples/UIExplorer/js/TextExample.ios.js

```diff
@@ -22,6 +22,9 @@
  */
 'use strict';
 
+const Platform = require('Platform');
+const isTVOS = (Platform && Platform.OS === 'ios' && Platform.SystemName === 'tvOS');
```

Comment:
Ah yes, unit tests may behave slightly differently. interfaceIdiom should be pretty reliable, it's Apple's recommended way of doing these types of checks: https://github.com/facebook/react-native/blob/master/React/Base/RCTPlatform.m#L24-L25